### DISCUSSION TITLE
Moving in the past: open solutions for data set design, spatial analysis and (spatial) statistical methods to investigate movement in Antiquity

### DISCUSSION THREAD

**Discussion by ddomizia** [archeofoss/archeofoss2022#3] [Panel proposals]
Panel proposal

## Moving in the past: open solutions for data set design, spatial analysis and (spatial) statistical methods to investigate movement in Antiquity

![rdr-lab-ago-d-erasmo](https://user-images.githubusercontent.com/39925492/158181924-60041d19-16ed-4ebe-b728-9c66de4f0337.jpg)

Image caption and credits
Heatmap representing the busiest streets in Rome in 1739 (2022 - Renata Ago and Domizia D'Erasmo)

Proponents:
- **Domizia D'Erasmo**
  - Sapienza University of Rome, Rome
  - [domizia.derasmo@uniroma1.it](mailto:domizia.derasmo@uniroma1.it)
  - [@domizzzia](https://twitter.com/domizzzia)
- **Noemi Giovino**
  - Sapienza University of Rome, Rome
  - [noemi.giovino@uniroma1.it](mailto:noemi.giovino@uniroma1.it)
  - [@noemigi](https://github.com/noemigi)

Using geographical open data is a steady method in archaeology, not so common is the use of movement-related data to investigate pathways and networks. This is probably due to the difficulty to source information and to represent phenomena regarding mobility in its space-time dimension. As known, archaeological data have mostly been treated in a static way to reconstruct ancient landscapes and are early considered as dynamically changing in space and time. On the other hand, methodologies for the study of the multiple aspects of mobility are extremely advanced in the contemporary world (e.g. improvement of transport; analysis of time travel measures to ensure the best practicability).

Nowadays data from current urban areas and large-scale mobility have increased considerably thanks to engineering and statistics scientific reports. What happens if we use these spatial and explorative analysis to query data sets on movement in the past? 
For archaeology the main trouble in applying these methods and analysis is the lack of open and purpose-built data sets. It is also interesting to investigate how and where this information can be obtained from. How far can we go in collecting movement data in archaeology? What is the most effective way of structuring data sets in order to “capture” movement?

The aim of this panel is to focus on the issue of open solutions for data sets design, algorithms for spatial analysis and geo-statistical methods to investigate the multifaceted phenomenon of movement in Antiquity. The panel also will welcome concrete case studies of investigation on historical and/or archaeological movement-related data by open-source spatial or (spatial) statistical analysis.

**Comment by benducke** [comment]
"Geostatistics" is a well-defined term that describes an apparatus of methods for spatial ressource prediction, with variogram analysis and kriging at its core: 

https://en.wikipedia.org/wiki/Geostatistics

The better term here might be "(spatial) statistical". I suggest changing the tile accordingly.

**Reply by ddomizia** [reply]
Thanks for the advice!
